src/server/saved_objects/service/create_saved_objects_service.js

```diff
@@ -31,60 +31,18 @@ import { SavedObjectsClient } from './saved_objects_client';
 // Eventually, we hope to build a first-class import / export API, at which point, we can
 // remove the migrator from the saved objects client and leave only document validation here.
 export function createSavedObjectsService(server, schema, serializer, migrator) {
-  const onBeforeWrite = async () => {
```

Comment:
is this meant to be removed in this PR?

Reply:
Yeah. It was breaking a number of tests, so I figured: I can either fix all of those tests to work with a thing that we want to remove, or I can just remove it and do it all in this PR.